### PR TITLE
Bah 5134 STEM Wizard UX Updates

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
Styling updates to the STEM Wizard from the UX Review of current staging content.

Updates were made for [5134](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/5134) and have been merged into master under a production flag so they would be visible on staging.

## Testing done
Testing passes locally and all changes were confirmed by the UX team.

## Screenshots
Screen Shot 2020-02-18 at 3 17 10 PM

## Acceptance criteria
- [x] "Apply Now" button needs to be sentence case
- [x] Content & bullet points should span full width of smaller displays, such as @320px
- [x] Period at end of "GI Bill eligibility page" link should not be linked or underlined.
- [x] Second line of text for 3rd STEM bullet point should left-align to first line of content. Looks like this is also an issue with the 2nd bullet point for smaller displays.
- [x] In scholarship content, "or" in 3rd bullet point should be bold.
- [x] In the middle bullet, the link wraps.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]

File: src/applications/static-pages/sass/modules/_m-education-wizard.scss [hunk 1]
```diff
@@ -49,7 +48,30 @@
...
+  input[type='radio'] {
     margin-left: -4rem;
   }
 }
+
+.wizard-edith-nourse-content {
+  @media (min-width: $medium-screen) {
+    margin-right: 64px;
+  }
+}
```
Comment (bkjohnson) [inline src/applications/static-pages/sass/modules/_m-education-wizard.scss]: The design system happens to include [responsive prefixes](https://design.va.gov/utilities/margins#responsive-prefixes), so you could use:

```html
<ul class="medium-screen:vads-u-margin-right--8" >
```

File: src/applications/static-pages/sass/modules/_m-education-wizard.scss [hunk 2]
```diff
@@ -49,7 +48,30 @@
...
+
+  @media (min-width: $small-screen) {
+    width: 90%;
+  }
+  width: 75%;
+}
+
+label {
+  display: block !important;
+}
```
Comment (bkjohnson) [inline src/applications/static-pages/sass/modules/_m-education-wizard.scss]: Were the labels being displayed some other way?

File: src/applications/edu-benefits/components/EducationWizard.jsx [hunk 3]
```diff
@@ -302,7 +313,7 @@ export default class EducationWizard extends React.Component {
             {newBenefit === 'extend' &&
             // Prod flag for 5134
             !environment.isProduction() ? (
-              <div className="vads-u-margin-right--8">
+              <div className="wizard-edith-nourse-content">
```
Comment (bkjohnson) [inline src/applications/edu-benefits/components/EducationWizard.jsx]: See previous comment

File: src/applications/edu-benefits/components/EducationWizard.jsx [hunk 4]
```diff
@@ -311,23 +322,26 @@ export default class EducationWizard extends React.Component {
                   </a>
                   , you must meet all the requirements below. You:
                 </strong>
-                <ul className="vads-u-margin-right--8">
+                <ul className="wizard-edith-nourse-content">
```
Comment (bkjohnson) [inline src/applications/edu-benefits/components/EducationWizard.jsx]: And again here

**Review by bkjohnson — COMMENTED** [review]
The `nowrap` and percentage width CSS changes are fine since the design system doesn't easily allow those, but one of the changes should be allowed by the design system.

File: src/applications/static-pages/sass/modules/_m-education-wizard.scss (hunk 2 above)
Comment (elder-joseph) [reply src/applications/static-pages/sass/modules/_m-education-wizard.scss]: Upon inspection of the label, it looked to be inheriting the style from: 
`errorable-radio-button label` 
Which was setting `display` to `inline-block;`, and caused styling issues conflicting with the design review comments provided by our UX team.

File: src/applications/static-pages/sass/modules/_m-education-wizard.scss (hunk 1 above)
Comment (elder-joseph) [reply src/applications/static-pages/sass/modules/_m-education-wizard.scss]: For a very visual example I tried using:
`<ul className="medium-screen:vads-u-color--orange">`

Which should change the color of the text of the `ul` to orange when the screen is at least `768px`, however this does not occur at any page size, unlike on other pages where I have used the responsive prefixes in the past.

**Review by bkjohnson — APPROVED** [review]
It's too bad that it sounds like there's a bug in the design system, but this seems fine.

**Comment by elder-joseph** [comment]
@bkjohnson Agreed, it is a little bit cleaner and easier to maintain using the design system class names vs individual css class names and definitions.

**Pull request merged**